### PR TITLE
[T] (quests) sinful_sorcery

### PR DESCRIPTION
**Pull request by nai200**
Co-authored-by: LANLAN <a1012881806@gmail.com>
Co-authored-by: winsky <winskyx@hotmail.com>
Co-authored-by: 青羽 <Wren_Xing@163.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/sinful_sorcery.snbt [hunk 1]
```diff
@@ -3,25 +3,25 @@
...
+			subtitle: "合成一个白粉笔"
 			description: [
-				"Most &5occultic magic&r involves &6rituals&r."
+				"大多数&5神秘魔法&r都包含&6仪式&r。"
 				""
-				"A major part of rituals involves drawing circles using &bChalk&r. This circle is called a &6Pentacle&r."
+				"其中，大部分仪式都会使用&b粉笔&r来画圈。而这种圈被称为&6五芒星&r。"
 				""
-				"Different rituals have different patterns that must be drawn; I can consult the &6Dictionary of Spirits&r for information on each."
+				"不同的仪式需要绘制不一样的图案；我可以查阅 &6魔灵全典&r来了解每一种仪式的资料。"
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/sinful_sorcery.snbt:22]:
```suggestion
				"不同的仪式需要绘制不一样的图案；我可以查阅&6魔灵全典&r来了解每一种仪式的资料。"
```

File: config/ftbquests/quests/chapters/sinful_sorcery.snbt [hunk 2]
```diff
@@ -3,25 +3,25 @@
...
+	title: "罪恶之魔法"
 	icon: "occultism:ritual_dummy/craft_miner_djinni_ores"
-	subtitle: ["Occultism"]
+	subtitle: ["神秘学"]
 	default_quest_shape: ""
 	default_hide_dependency_lines: false
 	quests: [
 		{
-			title: "Chalk Zone!"
+			title: "粉笔领域!"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/sinful_sorcery.snbt:13]:
```suggestion
			title: "粉笔领域!"
```

File: config/ftbquests/quests/chapters/sinful_sorcery.snbt [hunk 3]
```diff
@@ -146,14 +146,14 @@
...
 		{
-			title: "Bound in &mHuman&r Flesh"
+			title: "绑定在&m肉体&r之中"
 			x: -1.5d
 			y: -4.0d
-			subtitle: "Make the Dictionary of Spirits"
+			subtitle: "合成魔灵全典"
 			description: [
-				"&eOccultism&r, the art of &5binding spirits to my will&r. "
+				"&e神秘学&r，&5随我欲望绑定魔灵&r的艺术。 "
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/sinful_sorcery.snbt:154]:
```suggestion
				"&e神秘学&r，&5随我欲望绑定魔灵&r的艺术。"
```

File: config/ftbquests/quests/chapters/sinful_sorcery.snbt [hunk 4]
```diff
@@ -201,14 +201,14 @@
...
 			x: -1.5d
 			y: -2.0d
-			subtitle: "Make Otherstone"
+			subtitle: "合成一个异界石"
 			description: [
-				"&bOtherstone&r can be gathered by burning Andesite in the &6Spiritfire&r. How nostalgic!"
+				"&b异界石&r可以通过将安山岩投掷到&6灵火&r中净化来获得，这多么让人怀念呐！"
 				""
-				"It seems Otherstone comes from another dimension. That is, another, &onother&r, dimension. Perhaps the Spiritfire actually has some connection to it?"
+				"看来异界石来自另外一个维度。这就对了，另一个,&o另外一个&r，维度。也许灵火与它有一定的联系？"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/sinful_sorcery.snbt:211]:
```suggestion
				"看来异界石来自另外一个维度。这就对了，另一个，&o另外一个&r，维度。也许灵火与它有一定的联系？"
```

File: config/ftbquests/quests/chapters/sinful_sorcery.snbt [hunk 5]
```diff
@@ -349,22 +349,22 @@
...
+			subtitle: "击杀一个被附身的末影螨"
 			description: [
-				"Now that I've mastered the basics of spirit summons and bindings, it's time for me to push the limits."
+				"现在我已经掌握了魔灵召唤和绑定的基本知识，是时候让我挑战极限了。"
 				""
-				"I can &6cast out a spirit&r to bind it to an extradimensional entity, &6summoning it to me&r."
+				"我可以&6投出一个魔灵&r与超维生物绑定，&6并把它召唤到我身边&r。"
 				""
-				"This uses the &6Hedyrin's Lure&r pentacle. Upon starting the ritual, it will emit &6black particles&r. This means it is waiting for a living sacrifice: usually killing a mob, but in this case simply throwing an egg."
+				"这需要使用&6海德林诱惑术&r五芒星。 开始仪式后，它将发出&6黑色的粒子效果&r。这意味着它在等待一个活的祭品：通常是杀死一个生物，但在此仪式下只需要扔一个鸡蛋。"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/sinful_sorcery.snbt:363]:
```suggestion
				"这需要使用&6海德林诱惑术&r五芒星。开始仪式后，它将发出&6黑色的粒子效果&r。这意味着它在等待一个活的祭品：通常是杀死一个生物，但在此仪式下只需要扔一个鸡蛋。"
```

File: config/ftbquests/quests/chapters/sinful_sorcery.snbt [hunk 6]
```diff
@@ -608,28 +608,28 @@
...
-				"I need to remember I can only configure the Book while it is empty. I have to right-click it on a block to open its GUI."
+				"我需要记住，我只能配置空的召唤之书。并且需要对着一个方块右键才能打开它的图形用户界面。"
 				""
 				"{image:inferno:textures/quest/bookofcalling1.png width:183 height:100 align:1}"
-				"&6Set Deposit&r mode allows me to tell Janitors what block to insert items into. While in this mode, I can sneak right-click a block with the empty Book to set that block as the destination."
+				"&6设置存放位置&r模式允许我告诉清洁工应该将物品放到那个方块中。在这个模式中，我可以用空的召唤之书潜行右键一个方块来记录方块的坐标，将该方块设置为目的地。"
 				""
 				"{image:inferno:textures/quest/bookofcalling2.png width:183 height:100 align:1}"
-				"&6Set Base Location&r mode allows me to set the area the spirit works in. I can set the size in the GUI, and sneak right-click a block with the empty Book to set the center of the area."
+				"&6设置活动基点&r模式允许我设置魔灵的工作区域，我可以在召唤之书的用户图形界面里更改工作区域的大小，然后潜行右键一个方块以设置工作区域的中心。 "
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/sinful_sorcery.snbt:624]:
```suggestion
				"&6设置活动基点&r模式允许我设置魔灵的工作区域，我可以在召唤之书的用户图形界面里更改工作区域的大小，然后潜行右键一个方块以设置工作区域的中心。"
```

File: config/ftbquests/quests/chapters/sinful_sorcery.snbt [hunk 7]
```diff
@@ -759,17 +759,17 @@
 			x: 12.5d
 			y: -2.5d
 			shape: "hexagon"
-			subtitle: "Make a Dimensional Mineshaft"
+			subtitle: "建造一个维度矿井"
 			description: [
-				"The &bDimensional Mineshaft&r contains within it a &5portal to my home dimension&r! In a cruel twist of fate, however, it is &cmuch too small for me to fit through&r."
+				"&b维度矿井&r包含了一个前往&5主维度的传送门&r！然而，天意弄人，这个传送门&c小到我无法通过&r."
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/sinful_sorcery.snbt:764]:
```suggestion
				"&b维度矿井&r包含了一个前往&5主维度的传送门&r！然而，天意弄人，这个传送门&c小到我无法通过&r。"
```

**Review by Ricky-fight — COMMENTED** [review]
typo

**Pull request merged**